documentation/source/introduction.rst

```diff
@@ -29,6 +29,30 @@ the way:
 Many typical tokamak fusion reactor design optimisation problems are also implemented,
 leveraging the functionality of the above modules.
 
+How to use ``Bluemira``
+-----------------------
+
+``Bluemira`` is designed to be used by three different types of user.
+
+  Modeller
+    A Modeller will use a program given to them by a Reactor Designer to carry out design studies on a particular reactor design. Modellers will need to know about the parameters of a design and how to manipulate JSON files to modify those parameters.
+
+  Reactor Designer
+    A Reactor Designer will use ``Bluemira`` as a framework to create a design for a reactor. To design a reactor the design workflow strategy needs to be considered and codified. Using Designer and Builder objects each component of the reactor can be created and collected into a full reactor design or used individually.
```

Comment:
It's just that we haven't heard what a 'Reactor Designer' is yet, or what kind of 'program' the modeller is receiving. And, chronologically, the 'Reactor Designer' must do their job before the 'Modeller' can do theirs. 

I'm happy to keep this order, but we may need to clarify what 'program' means, as that's really explained in the next section.

Maybe this (again with a link to the 'Reactor Designer' definition, since the term isn't defined yet)?

> A Modeller will execute a reactor build workflow (created by a 'Reactor Designer'), to carry out studies on a reactor design. Modellers will need to know about the parameters of a design, and how to manipulate JSON files to modify those parameters.

Reply:
Yeah fair point. I’ll get that fixed up